Update Payment Facilities (#129)
mattermost/mattermost-handbook: company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md

If you are a staff member located in Canada or if your contract is with Open World Services \(OWS\), payroll will be made to you in accordance with OWS policy. For expense reimbursement, please follow the [OWS Reimbursement Process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#open-world-services-ows).

#### Rest of world

If you are a vendor, or a contractor located outside the United States and Canada, Mattermost requires a primary banking institution as a method of payment, as we use Bill.com for invoice processing and payment. If you are a staff member located outside of the United States, Canada, United Kingdom and Germany, you have the option of getting paid either by bill.com (via wire transfer) or TransferWise for invoice processing, expense reimbursement and payment. The currency in which payment is made is based on the currency stated within the agreement signed with Mattermost.

Primary Banking Institution and Bill.com

> The person or legal entity the bank account is registered to must be in the name of the contracting individual/entity.

Reviewer: ```suggestion
The person or legal entity the bank account is registered to will need to be in the name of the contracting individual/entity.
```

Reviewer: @ialorro - can you please check my suggestions and make sure that you agree? If you are good, I'll commit these

> _Example 1_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his surname, John Smith must have the following banking institution setup:

Reviewer: ```suggestion
_Example 1_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his surname, John Smith will need the following banking institution setup:
```

* The bank account is located in Spain.
* The bank account is his name or other known surname.

> _Example 2_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith must have the following banking institution setup:

Reviewer: ```suggestion
_Example 2_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith will need the following banking institution setup:
```

* The bank account is located in Spain.
* The bank account is in the name of Better Coding, LLC.

> If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through Bill.com. As noted above, Mattermost requires banking information from a primary banking institution to be able to make reimbursements through Bill.com. Approved expense reports are reimbursed either by the 15th or end of each month. 

Reviewer: ```suggestion
If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through Bill.com. As noted above, Mattermost needs banking information from a primary banking institution to be able to make reimbursements through Bill.com. Approved expense reports will follow our [Expense reimbursement policy](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#expense-reimbursement-policy).
```
